### DISCUSSION TITLE
Add Blogger Icon?

### DISCUSSION THREAD

**Discussion by rshmhrj** [jpanther/congo#70] [Ideas]
I had an [old blogger / blogspot.com site](http://dudewheysmehblog.blogspot.com/) and wanted to include it on my [new hugo/congo site](https://rshmhrj.io/), so I added a custom icon.

Normal: \
![image](https://user-images.githubusercontent.com/750874/147626030-c1d4a651-442f-4e01-aa18-b5b4352af09c.png)

On Hover: \
![image](https://user-images.githubusercontent.com/750874/147626048-f80709f7-7974-47d8-ad66-ce5cb17e6414.png)

@jpanther would you like to include it as part of congo?  If so, I can open a PR:
- blogger.svg file
- updated config.toml with author.links.blogger

TBH, the icon looks a little bigger than the others, so if you want it resized or if you think it doesn't make sense to include, totally understand!

**Comment by jpanther** [comment]
Thanks for the suggestion, the icons I'm using in the theme all come from FontAwesome. Have you tried using their [Blogger icon](https://fontawesome.com/v5.15/icons/blogger-b?style=brands)? You may find it scales more appropriately with the other icons.

**Reply by rshmhrj** [reply]
Yeah that definitely looks better: \
![image](https://user-images.githubusercontent.com/750874/147633501-3023efe4-aa5f-451d-906b-d37bbaa7ab12.png)

Thanks for your super speedy responses!  Please let me know if you want a PR for this -- happy to put it in if you think it makes sense.

**Reply by jpanther** [reply]
No problem, happy for you to submit a PR for this.

**Reply by rshmhrj** [reply]
Thanks @jpanther -- created [PR 71](https://github.com/jpanther/congo/pull/71)
